tests/component/_task_modules/channels/test_ai_channels.py

```diff
@@ -0,0 +1,722 @@
+import pytest
+
+from nidaqmx import Task
+from nidaqmx._task_modules.channels.ai_channel import AIChannel
+from nidaqmx.constants import (
+    AccelChargeSensitivityUnits,
+    AccelSensitivityUnits,
+    AccelUnits,
+    ACExcitWireMode,
+    BridgeConfiguration,
+    BridgeUnits,
+    ChargeUnits,
+    CJCSource,
+    CurrentShuntResistorLocation,
+    EddyCurrentProxProbeSensitivityUnits,
+    ExcitationSource,
+    ForceIEPESensorSensitivityUnits,
+    ForceUnits,
+    LVDTSensitivityUnits,
+    ResistanceConfiguration,
+    RTDType,
+    StrainGageBridgeType,
+    StrainGageRosetteMeasurementType,
+    StrainGageRosetteType,
+    TemperatureUnits,
+    TerminalConfiguration,
+    ThermocoupleType,
+    UsageTypeAI,
+    VoltageUnits,
+)
+
+
+# Note: Tests for other channel types will be less complete given that the underlying Python
+# implementation is code-generated and the underlying C API implementation is well unit-tested.
+@pytest.mark.parametrize(
```

Comment:
Does these multiple parametrize marks generate all the permutations? I think that might be overdoing it and a single flat parameter set is probably adequate. However, if it's on the order of 12 total cases then it doesnt' really matter.